tools/stress-cluster/services/Stress.Watcher/src/Extensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using k8s.Models;
+
+namespace k8s
```

Comment:
```suggestion
namespace Stress.Watcher
```

Reply:
I had thought that to provide extensions to the `V1Pod` type, I had to define them in the namespace in which it is defined, which is `k8s`?

Reply:
I don't know how those extensions are discovered so I cannot say for sure but this would be sort of an odd requirement unless they are looking for types explicitly in that namespace.